pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -525,27 +533,20 @@ def showable(self):
     @property
     def status(self):
         """ A short string describing the current status of storage setup. """
-        msg = _("No disks selected")
-
         if threadMgr.get(constants.THREAD_DASDFMT):
-            msg = _("Formatting DASDs")
+            return _("Formatting DASDs")
         elif flags.automatedInstall and not self.storage.root_device:
-            msg = _("Kickstart insufficient")
-        elif self.data.ignoredisk.onlyuse:
-            msg = P_(("%d disk selected"),
-                     ("%d disks selected"),
-                     len(self.data.ignoredisk.onlyuse)) % len(self.data.ignoredisk.onlyuse)
-
```

Comment:
This message is no more displayed intentionally I guess ?

Reply:
This message was never returned. It was always replaced with one of the messages below.

Reply:
Ah, I see..., okay.